frontend/src/types/Configuration.ts

```diff
@@ -23,15 +17,20 @@ export enum CurseSpawnType {
     Never = 'Never',
 }
 
+export enum Options {
+    NoneSelected = 0,
+    ConfigPerFloor = 1 << 0,
+    ConfigPerRoom = 1 << 1,
+    RemoveHealingItems = 1 << 2,
+    DisableMentorAbilities = 1 << 3,
+    DisableGiftOfIntern = 1 << 4,
+    DisablePinned = 1 << 5,
+    AwardSkillsPerLevel = 1 << 6,
+}
+
 export const DEFAULT_CONFIG: Configuration = {
     spawns: SpawnType.Looped,
     curseSpawns: CurseSpawnType.Randomly,
-    configPerFloor: false,
-    configPerRoom: false,
-    removeHealingItems: false,
-    disableMentorAbilities: false,
-    disableGiftOfIntern: false,
-    disablePinned: false,
-    awardSkillsPerLevel: false,
+    options: Options.RemoveHealingItems |  Options.DisablePinned,
     startingSkillIds: [],
```

Comment:
Is there any particular reason you wanted these as the default options or was this just for testing?

Reply:
Yes, just to make sure that it was loading it out of the default properly with the new changes - feel free to revert this post pull.